Carmunication/Controllers/SessionStart/SessionMapViewController.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  SessionMapViewController.swift
+//  Carmunication
+//
+//  Created by 김태형 on 2023/09/27.
+//
+
+import UIKit
+
+class SessionMapViewController: UIViewController {
```

Comment:
여기도 마찬가지로 final 클래스 붙여주는게 좋아보입니다!

```suggestion
final class SessionMapViewController: UIViewController {
```